command/loadbot/generator/deploy_generator.go

```diff
@@ -0,0 +1,70 @@
afc565b92915050565b600082601f8301126105e357600080fd5b81356105f384826020860161057f565b91505092915050565b60008135905061060b81610b13565b92915050565b6000806040838503121561062457600080fd5b6000610632858286016105bd565b9250506020610643858286016105fc565b9150509250929050565b60006020828403121561065f57600080fd5b600082013567ffffffffffffffff81111561067957600080fd5b610685848285016105d2565b91505092915050565b61069781610928565b82525050565b6106a68161093a565b82525050565b6106b58161094c565b82525050565b6106c48161095e565b82525050565b60006106d58261087a565b6106df8185610885565b93506106ef81856020860161097f565b6106f881610aeb565b840191505092915050565b61070c8161091e565b82525050565b6000604082019050610727600083018561068e565b610734602083018461069d565b9392505050565b600060408201905061075060008301856106ac565b61075d60208301846106bb565b9392505050565b6000602082019050818103600083015261077e81846106ca565b905092915050565b600060408201905081810360008301526107a081856106ca565b905081810360208301526107b481846106ca565b90509392505050565b600060608201905081810360008301526107d781866106ca565b905081810360208301526107eb81856106ca565b905081810360408301526107ff81846106ca565b9050949350505050565b600060208201905061081e6000830184610703565b92915050565b600061082e61083f565b905061083a82826109e4565b919050565b6000604051905090565b600067ffffffffffffffff82111561086457610863610abc565b5b61086d82610aeb565b9050602081019050919050565b600081519050919050565b600082825260208201905092915050565b60006108a18261091e565b91506108ac8361091e565b9250827fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff038211156108e1576108e0610a5e565b5b828201905092915050565b60006108f7826108fe565b9050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b6000819050919050565b60006109338261091e565b9050919050565b60006109458261091e565b9050919050565b60006109578261091e565b9050919050565b60006109698261091e565b9050919050565b82818337600083830152505050565b60005b8381101561099d578082015181840152602081019050610982565b838111156109ac576000848401525b50505050565b600060028204905060018216806109ca57607f821691505b602082108114156109de576109dd610a8d565b5b50919050565b6109ed82610aeb565b810181811067ffffffffffffffff82111715610a0c57610a0b610abc565b5b80604052505050565b6000610a208261091e565b91507fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff821415610a5357610a52610a5e565b5b600182019050919050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052602260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b6000601f19601f8301169050919050565b610b05816108ec565b8114610b1057600080fd5b50565b610b1c8161091e565b8114610b2757600080fd5b5056fe4368616e67696e67206772656574696e672066726f6d202725732720746f2027257327a2646970667358221220a4f650513c5038cc3c919651c8ca206cda3ae09ac470aebc8d96da7b43318d5e64736f6c634300080400334465706c6f79696e67206120477265657465722077697468206772656574696e673a"
```

Comment:
Why did you decide not to give users the ability to choose the contract to be deployed (using a json file for example) ?

Reply:
I love this idea 💯 

Added this functionality:

c4e1b40